app/models/conversion_host.rb

```diff
@@ -170,19 +170,24 @@ def get_conversion_log(path)
   end
 
   def check_conversion_host_role
-    playbook = "/usr/share/ovirt-ansible-v2v-conversion-host/playbooks/conversion_host_check.yml"
-    ansible_playbook(playbook)
+    playbook = "/usr/share/v2v-conversion-host-ansible/playbooks/conversion_host_check.yml"
```

Comment:
@fdupont-redhat was this path wrong before, or is a newer version installing to a new place?  Just wondering if we need to support both paths.

Reply:
@agrare The name of package containing the playbooks has changed, so did the playbooks path.

Reply:
Okay so it is possible that people have installed the old package and have the previous path?

Reply:
Nope. Because the path is on ManageIQ appliance, so if you upgrade the appliance, you have the new package in the correct path.

Reply:
:+1: